hw_diag/utilities/shell.py

```diff
@@ -13,9 +13,10 @@ def get_environment_var(diagnostics):
         'BALENA_APP_NAME',
         'FREQ',
         'FIRMWARE_VERSION',
-        'VARIANT'
+        'VARIANT',
+        'FIRMWARE_SHORT_HASH'
     ]
-    keys = ["BN", "ID", "BA", "FR", "FW", "VA"]
+    keys = ["BN", "ID", "BA", "FR", "FW", "VA", "FSH"]
```

Comment:
Not needed as mentioned above.

Reply:
Like above, I think we need to keep the short name until we remove the short naming scheme completely.

Reply:
@MuratUrsavas  - No new code we write relies on the short 2-character names and we don't have those for any new keys in endpoints either. See `/version` endpoint, it does not have any short names. Also the short names you see in `/initFile.txt` payload are only kept not to break existing code the uses them. So if you're adding a new short name, by definition no code is using it hence there is no reason for it to be present.

Reply:
OK, this one is easy. I can remove it. Let's keep the other one for the reason I've mentioned earlier. Cool?

Reply:
Although, I still prefer to keep all of them just for consistency and remove them later altogether.